--- a/Do/src/Do.UI/KeybindingTreeView.cs	2009-08-08 22:41:51 +0000
+++ b/Do/src/Do.UI/KeybindingTreeView.cs	2009-11-04 17:05:23 +0000
@@ -71,7 +71,7 @@
 
 			foreach (KeyBinding binding in Services.Keybinder.Bindings) {
 				ks = (string.IsNullOrEmpty (binding.KeyString)) ? Catalog.GetString ("Disabled") : binding.KeyString;
-				store.AppendValues (binding.Description, ks, binding.DefaultKeyString, binding);
+				store.AppendValues (Catalog.GetString(binding.Description), ks, binding.DefaultKeyString, binding);
 			}
 		}
 		

