=== modified file 'Do/src/Do.Core/Controller.cs'
--- a/Do/src/Do.Core/Controller.cs	2009-09-10 07:32:34 +0000
+++ b/Do/src/Do.Core/Controller.cs	2009-11-04 16:57:04 +0000
@@ -137,46 +137,46 @@
 				OnSummonKeyPressEvent, true));
 			
 			// this keybinding is disabled by default - note the empty keybinding
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Summon in Text Mode"), "",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Summon in Text Mode", "",
 				OnTextModeSummonKeyPressEvent, true));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Enter Text Mode"), "period",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Enter Text Mode", "period",
 				OnTextModePressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Clear"), "Escape",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Clear", "Escape",
 				OnClearKeyPressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Copy to Clipboard"), "<Control>c",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Copy to Clipboard", "<Control>c",
 				OnCopyEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Paste from Clipboard"), "<Control>v",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Paste from Clipboard", "<Control>v",
 				OnPasteEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Previous Pane"), "<Shift>Tab",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Previous Pane", "<Shift>Tab",
 				OnPreviousPanePressEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Next Pane"), "Tab",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Next Pane", "Tab",
 				OnNextPanePressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Previous Item"), "Up",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Previous Item", "Up",
 				OnPreviousItemPressEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Next Item"), "Down",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Next Item", "Down",
 				OnNextItemPressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("First Item"), "Home",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("First Item", "Home",
 				OnFirstItemPressEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Last Item"), "End",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Last Item", "End",
 				OnLastItemPressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Previous 5 Results"), "Page_Up",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Previous 5 Results", "Page_Up",
 				OnNextItemPagePressEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Next 5 Results"), "Page_Down",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Next 5 Results", "Page_Down",
 				OnPreviousItemPagePressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Step Out of Item"), "Left",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Step Out of Item", "Left",
 				OnStepOutItemPressEvent));
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Step Into Item"), "Right",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Step Into Item", "Right",
 				OnStepInItemPressEvent));
 			
-			Services.Keybinder.RegisterKeyBinding (new KeyBinding (Catalog.GetString ("Select Multiple Items"), "comma",
+			Services.Keybinder.RegisterKeyBinding (new KeyBinding ("Select Multiple Items", "comma",
 				OnSelectionKeyPressEvent));
 		}

